source/Octopus.Tentacle.Contracts/Services/Capabilities/IAsyncCapabilitiesServiceV2.cs

```diff
@@ -4,10 +4,13 @@
 using Halibut.Transport.Caching;
 using Octopus.Tentacle.Contracts.Capabilities;
 
+// Don't "fix" this namespace, this is what we originally did and perhaps
+// what we need to keep doing. 
 namespace Octopus.Tentacle.Services.Capabilities
```

Comment:
If changing this to what rider wants it to be works and the test passes then it should be safe to do as we have backwards compatibility testing. Also I don't think package location matters for services.

Reply:
But also this is the most as-is change as possible